Poisonous-Potato/JTK-Website
# Add Custom Styling and Components

### Description
Create custom styling and components

### How to Test
1. Run the portal
2. Check in landing page

### Screenshot
![image](https://user-images.githubusercontent.com/39304683/168429665-bf00aac2-001b-4b51-9eae-6138eaee6b93.png)
![image](https://user-images.githubusercontent.com/39304683/168429681-ea093b99-b512-4e9d-be78-1e343d6c60c4.png)

### Issues
Closes #3
